js/extension/components/FeatureReports.jsx

```diff
@@ -65,11 +61,7 @@ class FeatureReports extends React.Component {
             .getReports(this.state.feature_id, this.state.layer_id)
             .subscribe((reports) => 
                 {
-                    let reports_parsed = reports;
-                    if (! new URLSearchParams(window.location.search).has("usemocks")) { 
-                        reports_parsed = reports.map((report) => JSON.parse(report));
-                    } 
-                    this.setState({ reports : reports_parsed })
+                    this.setState({ reports : reports.map((report) => JSON.parse(report)) })
```

Comment:
looks strange to me to have to do this here.
Should be 
```suggestion
                    this.setState({ reports })
```
With the work done in the service, which output should be clean.

Reply:
yes you're right. that's fixed.